src/store/proposals/sagas/addProposal.test.ts

```diff
@@ -0,0 +1,557 @@
+        ])
+        .select(governorAdminSelector)
+        .call(
+          mockGovernorContract.propose,
+          [proposalGovernorAdminPayload.payload.Values[0].Target],
+          [proposalGovernorAdminPayload.payload.Values[0].Value],
+          [proposalGovernorAdminPayload.payload.Values[0].Signature],
+          [proposalGovernorAdminPayload.payload.Values[0].Calldata],
+          proposalGovernorAdminPayload.payload.Description
+        )
+        .hasFinalState(successState)
+        .run();
+    });
+
+    it('when wallet is not connected', async () => {
+      await getBasePath()
+        .provide([
+          [matchers.select(accountSelector), undefined],
+          [matchers.select(governorAdminSelector), mockGovernorContract],
+          [
+            matchers.call(
+              mockGovernorContract.propose,
+              [proposalGovernorAdminPayload.payload.Values[0].Target],
+              [
+                proposalGovernorAdminPayload.payload.Values[0].Value,
+              ] as BigNumberish[],
+              [proposalGovernorAdminPayload.payload.Values[0].Signature],
+              [proposalGovernorAdminPayload.payload.Values[0].Calldata],
+              proposalGovernorAdminPayload.payload.Description
+            ),
+            mockTx,
+          ],
+          [matchers.call(mockTx.wait), mockTxReceipt],
+        ])
+        .select(governorAdminSelector)
+        .hasFinalState(failureWalletNotConnectedState)
+        .run();
+    });
+
+    it('fetching error', async () => {
+      await getBasePath()
+        .provide([
+          [matchers.select(accountSelector), testAccount],
+          [matchers.select(governorAdminSelector), mockGovernorContract],
+          [
+            matchers.call(
+              mockGovernorContract.propose,
+              [proposalGovernorAdminPayload.payload.Values[0].Target],
+              [
+                proposalGovernorAdminPayload.payload.Values[0].Value,
+              ] as BigNumberish[],
+              [proposalGovernorAdminPayload.payload.Values[0].Signature],
+              [proposalGovernorAdminPayload.payload.Values[0].Calldata],
+              proposalGovernorAdminPayload.payload.Description
+            ),
+            throwError(),
+          ],
+          [matchers.call(mockTx.wait), mockTxReceipt],
+        ])
+        .select(governorAdminSelector)
+        .call(
+          mockGovernorContract.propose,
+          [proposalGovernorAdminPayload.payload.Values[0].Target],
+          [proposalGovernorAdminPayload.payload.Values[0].Value],
+          [proposalGovernorAdminPayload.payload.Values[0].Signature],
+          [proposalGovernorAdminPayload.payload.Values[0].Calldata],
+          proposalGovernorAdminPayload.payload.Description
+        )
+        .hasFinalState(failureState)
+        .run();
+    });
+  });
+  describe('checkAddEligibility', () => {
+    const proposalGovernorAdminPayload: ProposalsActions['addProposal'] = {
```

Comment:
I think something is not right here. You're creating a redux action and then use it as a mock result for the `userProposalsListQuery`. I'm not sure why it's not throwing any errors, but I think it should be changed. Please change the name and the type to `ProposalListQueryResult`

Reply:
in checkAddEligibility I'm using userProposalsListQuery, not ProposalListQueryResult. I put userProposalsListQuery to proposalListQuery to keep similar queries in one place.

As you wrote above, I change to mockAddProposalValues and moved above both describe.